.github/workflows/build.yml

```diff
@@ -151,7 +151,7 @@ jobs:
           python-version: 3.8
       - name: Setup pip (base)
         run: |
-          pip install -U pip setuptools wheel
+          pip install --user -U pip setuptools wheel
```

Comment:
Alternatively we could also add this `--user` flag just for this job since it's the one that was failing?

And the others run on Ubuntu. Or do you think it's better to have `--user` everywhere for consistency?

Reply:
>  everywhere for consistency?

yeah, until we get better reuse mechanisms (ahem, yaml anchors)  i'm inclined to keep everything the same as possible. heck, i almost added a matrix to to all the singleton jobs for this reason.

Reply:
:+1: